parquet/src/main/java/org/apache/iceberg/parquet/VectorizedParquetReader.java

```diff
@@ -48,11 +49,12 @@
   private boolean reuseContainers;
   private final boolean caseSensitive;
   private final int batchSize;
+  private final NameMapping nameMapping;
 
   public VectorizedParquetReader(
-      InputFile input, Schema expectedSchema, ParquetReadOptions options,
-      Function<MessageType, VectorizedReader<?>> readerFunc,
-      Expression filter, boolean reuseContainers, boolean caseSensitive, int maxRecordsPerBatch) {
+      InputFile input, Schema expectedSchema, ParquetReadOptions options, Function<MessageType,
+      VectorizedReader<?>> readerFunc, NameMapping nameMapping, Expression filter, boolean reuseContainers,
```

Comment:
Please don't split types across lines. I think moving `Function<MessageType` to the previous line should be reverted.